schematic/models/validate_manifest.py

```diff
@@ -244,10 +244,19 @@ def validate_manifest_values(self, manifest, jsonSchema
             for error in sorted(v.iter_errors(annotation), key=exceptions.relevance):
                 errorRow = i + 2
                 errorCol = error.path[-1] if len(error.path) > 0 else "Wrong schema"
```

Comment:
Not sure if errorCol here is still needed

Reply:
It is, it should be added as part of the return error list, like the [other errors](https://github.com/Sage-Bionetworks/schematic/blob/7df6950c248c90904bb7214b56cb25d7354f932b/schematic/models/validate_attribute.py#L70) generated

Reply:
@GiaJordan does the order matter? If I have `validate_manifest_values` function returns `[errorRow, errorColName, errorCol, errorMsg, errorVal]`, does that work?

Reply:
When you append the list of stings to the list of errors, I don't think `errorColName` should be included because whatever accesses the error list later likely does so by index and that could throw it off. But the name of the attribute can be passed to `generate_schema_error` to be displayed in the message

Reply:
Okay. That makes sense.